code/__DEFINES/experisci.dm

```diff
@@ -8,8 +8,6 @@
 /// Integer stages, should be whole numbers with total being included
 /// to support rendering ``value of total``, or something akin to it.
 #define EXPERIMENT_INT_STAGE "integer"
-/// Float stages, the value should be between 0 and 1 representing percent completion
-#define EXPERIMENT_FLOAT_STAGE "float"
```

Comment:
This one is also probably safe to keep, should people want to make float experiments in the future